Sources/DataPipeline/CustomerIO+Events.swift

```diff
@@ -11,10 +11,12 @@ public extension CustomerIO {
     /// Associate a user with their unique ID and record traits about them.
     /// - Parameters:
     ///   - traits: A dictionary of traits you know about the user. Things like: email, name, plan, etc.
+    @available(*, deprecated, message: "Use 'identify(userId:traits:)' with [String: Any] traits parameter instead. Support for Codable traits will be removed in a future version.")
```

Comment:
This method was mainly for attaching profile traits to anonymous profiles without requiring `userId`. Since `identify` needs a `userId`, I don't think it's a direct replacement. Should this now be handled by `setProfileAttributes` instead?